### DISCUSSION TITLE
Browser support of swiper

### DISCUSSION THREAD

**Discussion by jonybekov** [nolimits4web/swiper#6683] [General]
Hi, everyone. What is the browser compatibility of swiper 9?
It seems like a great fit for my Smart TV project which uses chromium 53+.
If that kind of old browsers are not supported by latest swiper even with polyfills, which version I can use?

**Comment by jimhaakman** [comment]
Hi @jonybekov, I have the same question. What version (and/or polyfills) did you end up using? Thanks 🙏
